scripts/add-refs-scpca-meta.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
```

Comment:
Could use a quick comment here, something like...

```suggestion
"""
If missing, add `ref_mito` and `ref_fasta_index` fields to `scpca-meta.json` checkpoint files in S3.
"""
```

Reply:
Woops, back for one more comment I forgot! We should also include some usage info up here, in particular for providing non-default args. Thinking of this comment you made in the PR - 

> I have been testing using `scpca/processed/checkpoints`, but when we run this for real, we will want to use `scpca_prod/checkpoints`